mgmt-hub/deploy-mgmt-hub.sh

```diff
@@ -447,37 +497,73 @@ if isMacOS; then
             fatal 2 "the commands jq and envsubst are required, and since brew is not installed, we can not install them for you"
         fi
     fi
-else   # ubuntu
-    echo "Updating apt package index..."
-    runCmdQuietly apt-get update -q
+else   # ubuntu and redhat
+    echo "Updating ${PKG_MNGR} package index..."
+    runCmdQuietly ${PKG_MNGR} update -q -y
     echo "Installing prerequisites, this could take a minute..."
-    runCmdQuietly apt-get install -yqf jq gettext-base make
+    runCmdQuietly ${PKG_MNGR} ${PKG_MNGR_INSTALL_QY_CMD} jq ${PKG_MNGR_GETTEXT} make
 
     # If docker isn't installed, do that
-    if ! command -v docker >/dev/null 2>&1; then
+    if ! isCmdInstalled docker; then
         echo "Docker is required, installing it..."
-        curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
-        chk $? 'adding docker repository key'
-        add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
-        chk $? 'adding docker repository'
-        apt-get install -y docker-ce docker-ce-cli containerd.io
-        chk $? 'installing docker'
+        if isUbuntu18; then
+            curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+            chk $? 'adding docker repository key'
+            add-apt-repository "deb [arch=${ARCH_DEB}] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+            chk $? 'adding docker repository'
+            if [ "${ARCH}" = "amd64" ]; then
+                ${PKG_MNGR} install -y docker-ce docker-ce-cli containerd.io
+            elif [ "${ARCH}" = "ppc64le" ]; then
+                ${PKG_MNGR} install -y docker-ce containerd.io
+	        else
+	            fatal 1 "hardware plarform ${ARCH} is not supported yet"
+            fi
+            chk $? 'installing docker'
+        else # redhat
+            EPEL_VERSION=7
+            DOCKER_VERSION=19.03.8
+            rpm -i --force https://dl.fedoraproject.org/pub/epel/epel-release-latest-${EPEL_VERSION}.noarch.rpm
+            chk $? 'installing epel-release-latest-${EPEL_VERSION}.noarch.rpm'
+            ${PKG_MNGR} install -y containerd
+            chk $? 'installing containerd'
+            rpm -i --force https://oplab9.parqtec.unicamp.br/pub/ppc64el/docker/version-${DOCKER_VERSION}/centos/docker-ce-cli-${DOCKER_VERSION}-3.el${EPEL_VERSION}.ppc64le.rpm
```

Comment:
Why the `--force` flag?

Reply:
I used `--force` flag because it allows successfully reinstall `RPM` package if it is existed in the OS by some reason. Especially it is important and true about `EPEL` package. If I remove this flag `rpm` and `yum` installers will return `false (1)` for existing `RPM` package.